.gitignore

```diff
@@ -0,0 +1 @@
+*.png
```

Comment:
Ignoring all *.png is probably not what we want here, espectially this is the root gitignore. 
If your code are creating local image files, consider create these images under something like `.tmp` directory and gitignore .tmp. Also move this gitignore under the `cross-edits-analysis`.